pkg/synchromanager/clustersynchro/cluster_synchro.go

```diff
@@ -441,8 +441,9 @@ func (s *ClusterSynchro) clusterStatusUpdater() {
 
 	for range s.updateStatusCh {
 		status := s.genClusterStatus()
+		klog.V(2).InfoS("Update cluster ready condition and sync resources status", "cluster", s.name, status.Conditions[0])
```

Comment:
I think the log should still be printed in the `UpdateClusterStatus`, because there is really no need to print the log when the state has not changed.

Too many repeated update logs are not good for troubleshooting.